sci-libs/symmetrica/symmetrica-2.0.ebuild

```diff
@@ -31,12 +21,10 @@ src_prepare() {
 #	plus documentation that you can use as you wish in your programs.
 #	For sage and ease of use we make it into a library with the following
 #	makefile (developped by F. Bissey and T. Abbott (sage on debian).
-	cp "${FILESDIR}/makefile" "${S}/makefile"
+	cp "${FILESDIR}/makefile" "${S}/makefile" || die
```

Comment:
I think, we should merge this request first and merge the changes from the sage-on-gentoo overlay in a 2nd one.